.idea/vcs.xml

```diff
@@ -2,5 +2,7 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/ivy_tests/test_array_api" vcs="Git" />
```

Comment:
Hey! I don't think you should commit changes in this file. Can you remove this from the commit.